notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                return "Feature layer: {} not found!".format(feature_layer)
</pre><p>I would raise an exception here too and on lines 17 and 19</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>